scripts/qa_crowbarsetup.sh

```diff
@@ -1298,24 +1298,25 @@ function onadmin_setup_local_zypper_repositories
     zypper lr -e - | sed -n '/^name=/ {s///; /ptf/! p}' | \
         xargs -r zypper rr
 
-    uri_base="http://${clouddata}${clouddata_base_path}"
+    uri_base=$smturl
```

Comment:
This switches the default from `$clouddata` to `$susedownload`.
This can work for sure, but then we should discuss the need to mirror these repositories on cloudata. Otherwise we need to configure all our jobs to set the smturl variable.

@bmwiedemann what is your take on this?